modules/oadp-release-notes-1-1-4.adoc

```diff
@@ -27,14 +45,19 @@ The following bugs have been fixed in this release:
 * link:https://issues.redhat.com/browse/OADP-1390[OADP-1390]
 * link:https://issues.redhat.com/browse/OADP-1650[OADP-1650]
 * link:https://issues.redhat.com/browse/OADP-1487[OADP-1487]
-
+////
 
 [id="known-issues1.1.4_{context}"]
 == Known issues
 
 This release has the following known issues:
 
-* OADP backups might fail because a UID/GID range might have changed on the cluster where the application has been restored, with the result that OADP does not back up and restore {product-title} UID/GID range metadata. To avoid the issue, if the backed application requires a specific UUID, ensure the range is available when restored. An additional workaround is to allow OADP to create the namespace in the restore operation.
+.OADP backups might fail because a UID/GID range might have changed on the cluster
+
+OADP backups might fail because a UID/GID range might have changed on the cluster where the application has been restored, with the result that OADP does not back up and restore {product-title} UID/GID range metadata. To avoid the issue, if the backed application requires a specific UUID, ensure the range is available when restored. An additional workaround is to allow OADP to create the namespace in the restore operation.
+
+.A restoration might fail if ArgoCD is used during the process due to a label used by ArgoCD
 
-* A restoration might fail if ArgoCD is used during the process due to a label used by ArgoCD, `app.kubernetes.io/instance`. This label identifies which resources ArgoCD needs to manage, which can create a conflict with OADP's procedure for managing resources on restoration. To work around this issue, set `.spec.resourceTrackingMethod` on the ArgoCD YAML to `annotation+label` or `annotation`. If the issue continues to persist, then disable ArgoCD before beginning to restore, and enable it again when restoration is finished.
+A restoration might fail if ArgoCD is used during the process due to a label used by ArgoCD, `app.kubernetes.io/instance`. This label identifies which resources ArgoCD needs to manage, which can create a conflict with OADP's procedure for managing resources on restoration. To work around this issue, set `.spec.resourceTrackingMethod` on the ArgoCD YAML to `annotation+label` or `annotation`. If the issue continues to persist, then disable ArgoCD before beginning to restore, and enable it again when restoration is finished. 
```

Comment:
It would be nice if we add bug link here

Reply:
I could not find the bug to link to.
It is listed in the FAQs - https://access.redhat.com/articles/5456281#oadp-restore-fails-with-argocd-29
But I could not be certain of the corresponding Jira ticket